test/e2e/network/netpol/network_policy.go

```diff
@@ -159,39 +158,9 @@ var _ = SIGDescribeCopy("Netpol [LinuxOnly]", func() {
 		})
 
 		ginkgo.It("should support a 'default-deny-all' policy [Feature:NetworkPolicy]", func() {
-			np := &networkingv1.NetworkPolicy{}
```

Comment:
these changes seem to be unrelated to the new test case , should they be in a separate PR ?

Reply:
Let me split them

Reply:
https://github.com/kubernetes/kubernetes/pull/97638